dev-docs/how-prebid-on-amp-works.md

```diff
@@ -13,33 +13,33 @@ nav_section: prebid-amp
 # How Prebid on AMP Works
 {: .no_toc}
 
-{% include amp-deprecation-warning.md %}
+This page has information about how Prebid works with AMP to show Prebid ads on AMP pages.
 
-This page has information about how Prebid.js works with AMP to show Prebid ads on AMP pages.
+The diagram below shows how the pieces fit together:
 
-The diagram below shows how the pieces fit together.  In the list below, the items with numbers correspond with the numbered actions in the diagram.
+1. AMP runtime calls Prebid Server URL, including macros.
 
-1. The content page (`publisher.com/amp_page.html` in the diagram) is just a normal AMP page that serves over HTTPS
-    + It has a `meta` element pointing to the cross-domain host where Prebid.js runs (in `remote.html`).  It needs to be a cross-domain host so that Prebid.js can execute in a secure environment that doesn't affect the AMP page.
-    + It uses the `amp-ad` element, which loads an iframe from a different domain (which is what we call the "cross-domain host" below).  This element contains some JSON configuration info which is sent to the cross-domain host.
+2. Prebid Server retrieves demand partner configuration.
 
-2. The creative (`creative.html` in the diagram) is what your ad ops team loads into your ad server.  It's a mostly standard Prebid.js creative with a few tweaks to allow it to work in AMP-land, and it's what serves into the iframe defined by the `amp-ad` element.
+3. Prebid Server sends bid requests to demand partners and caches bid responses.
 
-3. When the creative is delivered into the `amp-ad` element, it posts a message to the cross-domain host (`remote.html`) asking for Prebid creatives.
+4. Prebid Server responds to AMP runtime with key-value targeting.
 
-4. The cross-domain host (`amp.publisher.com/remote.html` in the diagram) runs a Prebid auction, fetching creatives for each of the ad slots defined in the content page.
+5. AMP Network constructs ad request URL and Prebid creative is served.
 
-5. The cross-domain host, having run the auction across N bidders and fetched creative content for each ad slot, posts a message back to the creative with the actual content to render.
+6. Creative content is retrieved from cache and renders.
 
-6. The creative renders the creative content it receives from the cross-domain host, and an ad is shown to the user.
-
-For more information, see [the AMP example in our Github repo](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp) (which uses the same filenames as in the diagram below), and the **Related Topics** list at the bottom of this page.
-
-![Prebid AMP Overview Diagram]({{site.github.url}}/assets/images/dev-docs/prebid-amp.png)
+{: .pb-img.pb-md-img :}
+![Prebid AMP Overview Diagram]({{site.github.url}}/assets/images/dev-docs/amp-rtc.png)
```

Comment:
Can we make this image larger?  It's a bit hard to read on the page without enlarging to 150%.

Let's also move this image to the top and have the step-by-step description come after.